test/compress/collapse_vars.js

```diff
@@ -1209,3 +1207,42 @@ collapse_vars_regexp: {
         })();
     }
 }
+
+keep_assign: {
```

Comment:
Would you like me to copy the whole section below each of the two original blocks of tests, or just the affected case?